src/zenml/zen_stores/migrations/versions/03caabf47fc5_added_logs_table.py

```diff
@@ -0,0 +1,62 @@
+"""Added logs table [03caabf47fc5].
+
+Revision ID: 03caabf47fc5
+Revises: 9971237fa937
```

Comment:
Before you merge you should adjust this to point to the current revision head on develop, e.g. `0.39.1` right now
```suggestion
Revises: 0.39.1
```